### DISCUSSION TITLE
Centering short codes, specifically YouTube

### DISCUSSION THREAD

**Discussion by axjms1** [willfaught/paige#31] [Q&A] [answered]
Hello Will,
Sorry to be a pest but is there a way to center the contents of a shortcode?  I am hoping to center a youtube video specifically.

This works in my markdown file...

div style="text-align: center;"

![alt text](https://markdown.land/wp-content/uploads/2021/06/markdown-512px.png "Our logo")

div

This doesn't...

div style="text-align: center;"
{{< paige/youtube maxwidth="50%" controls=true description="blah blah" video="videoid">}}
div

Any ideas?  I pulled off the opening and closing brackets to stop my question from rendering.

**Comment by willfaught** [comment]
Your HTML isn't formatting correctly, so I might be missing something. I think you want to align it with flexbox instead of text-align. Have you tried wrapping paige/youtube in paige/figure? See https://willfaught.com/paige/shortcodes/youtube/#figure for an example

**Reply by axjms1** [reply]
I will try the figure thing, Thank you.  I was trying to just show the code in my question and it kept rendering the tags.

**Reply by axjms1** [reply]
I tried to wrap it in your figure short code and I tried to wrap it in the gallery shortcode as well.   Neither worked.  It appears centered if the video is huge but I would like to open it at about 50%.  Thank you.

**Reply by willfaught** [reply, marked as answer]
I forgot to say that videos are automatically sized to be the width of the outermost Bootstrap container, so there's no way to make them smaller width-wise without turning that off. If you want to do so, you'll want to override https://github.com/willfaught/paige/blob/master/layouts/partials/paige/scripts.html in your site and comment out [these lines](https://github.com/willfaught/paige/blob/master/layouts/partials/paige/scripts.html#L28-L29). However, you'll then have to set the width manually for all videos.

Note that setting the width to 50% will also apply to mobile layouts, which might not be what you want.

**Reply by axjms1** [reply]
Thank you for your help.
